src/inotify.rs

```diff
@@ -108,7 +108,7 @@ impl EventLoop {
             &evented_inotify,
             INOTIFY,
             mio::Ready::readable(),
-            mio::PollOpt::edge(),
+            mio::PollOpt::level(),
```

Comment:
might want to add a comment here, referencing the issue

Reply:
Sure.